pkg/controller/gittrack/gittrack_controller.go

```diff
@@ -536,6 +527,9 @@ func (r *ReconcileGitTrack) ignoreObject(u *unstructured.Unstructured, owner far
 		} else if ownerIsClusterGittrack && r.clusterGitTrackMode == farosflags.CGTMExcludeNamespaced {
 			return true, "namespaced resources cannot be managed by ClusterGitTrack", nil
 		}
+	} else if ownerIsGittrack {
+		// cluster scoped object managed from namespaced gittrack. Disallow
+		return true, "cluster scoped object managed from gittrack are not allowed", nil
```

Comment:
We should capitalise `GitTrack` to be consistent, I'm also tempted to reorder the phrasing slightly, WDYT?
```suggestion
		return true, "a GitTrack cannot manage a cluster scoped resource", nil
```